core/src/smartcontracts/isi/domain.rs

```diff
@@ -339,6 +359,9 @@ pub mod isi {
             } = self;
 
             let _ = state_transaction.world.account(&source_id)?;
+            // Exceptionally, the destination account should not be recognized here.
+            // Otherwise, the risk is that the previous owner can no longer activate the current owner who cannot yet take any action by oneself.
+            // Thus, the destination account should be explicitly registered before this transfer
             let _ = state_transaction.world.account(&destination_id)?;
```

Comment:
It may be dangerous to recognize an account when targeted as `Transfer<Account, DomainId, Account>` destination. The risk is that the previous owner can no longer activate the current owner who cannot yet take any action by oneself